src/pages/chart-view/chart-view.scss

```diff
@@ -0,0 +1,26 @@
+@import "../../styles/mixins.scss";
+@import "../../styles/variables.scss";
+
+main-app-element {
+  overflow-y: scroll;
+}
+
+chart-view {
+  width: 100%;
+  align-self: center;
+
+  chart-visualization {
+    display: block;
+    height: 85vh;
```

Comment:
Use rem?

Reply:
We didn't add this.  It was just moved.  My guess is that we chose vh because we want the height relative to the entire view port.